src/CSSBattle/93_GreatWall/WeiPin.html

```diff
@@ -0,0 +1,78 @@
+<!-- 99.5% -->
+<div class='cir'>
+    <div class='ycir'></div>
+      <div class='property stripe1'></div>
+      <div class='property stripe2'></div>
+      <div class='property stripe3'></div>
+      <div class='property stripe4'></div>
+      <div class='property stripe5'></div>
+      <div class='property stripe6'></div>
+      <div class='connector stripe7'></div>
+      <div class='connector stripe8'></div>
+      <div class='connector stripe9'></div>
+      </div>
+    <style>
+      body {
+        background: #4F77FF;
+        justify-content: center;
+        align-items: center;
+        display: flex;
+      }
+      .cir {
+        width: 200px;
+        height: 200px;
+        background: #191919;
+        border-radius: 100%;
+        overflow: hidden;
+       
+      }
+      .ycir {
+        background:#F9E492;
+        width: 40px;
+        height: 40px;
+        border-radius: 100%;
+        margin: 40 40
+      }
+      .property {
+        background: #F9E492;
+        width: 100px;
+        height: 16px;
+      }
+      .connector {
+        background: #D6B73F;
+        width: 20px;
+        height: 16px;
+        transform: skewY(-33deg) translateY(-6.5px);
+      }
+      .stripe1 {
+        margin-top: 41;
+      }
+      .stripe2 {
+        margin-top: 10;
+      }
+      .stripe3 {
+        margin-top: 10;
+      }
+      .stripe4 {
+        float: right;
+        margin: -80 -20
+      }
+      .stripe5 {
+        float: right;
+        margin: -54 -20
+      }
+      .stripe6 {
+        float: right;
+        margin: -29 -20
+      }
+      .stripe7 {
```

Comment:
can look into box-shadow for repeating elements. a neat trick i learnt from the internet!!!